src/plugins/topbar/topbar.jsx

```diff
@@ -19,14 +20,49 @@ export default class Topbar extends React.Component {
     this.setState({url: value})
   }
 
+  loadSpec = (url) => {
+    this.props.specActions.updateUrl(url)
+    this.props.specActions.download(url)
+  }
+
+  onUrlSelect =(e)=> {
+    let url = e.target.value || e.target.href
+    this.loadSpec(url)
+    e.preventDefault()
+  }
+
   downloadUrl = (e) => {
-    this.props.specActions.updateUrl(this.state.url)
-    this.props.specActions.download(this.state.url)
+    this.loadSpec(this.state.url)
     e.preventDefault()
   }
 
+  componentWillMount() {
```

Comment:
This is so you can link to a spec with `?name=Spec%20Name`. I didn't notice all the configs get merged. I've updated to use getConfigs and documented in the README better.